schema/collections/dataset.json

```diff
@@ -70,7 +70,8 @@
         "R/P0.5M",
         "R/P4M",
         "R/P1W",
-        "R/PT1H"
+        "R/PT1H",
+        "Irregular"
```

Comment:
As per https://resources.data.gov/resources/dcat-us/#accrualPeriodicity the actual value should be lowercase, we should just capitalize it in the display names. Will fix.